Sources/MapboxSearchUI/SearchTextField.swift

```diff
@@ -38,7 +46,7 @@ class SearchTextField: UIView {
         textField.adjustsFontForContentSizeCategory = true
         iconView.adjustsImageSizeForAccessibilityContentSizeCategory = true
         
-        textField.attributedPlaceholder = NSAttributedString(string: Strings.SearchTextField.placeholder,
+        textField.attributedPlaceholder = NSAttributedString(string: customPlaceholder ?? Strings.SearchTextField.placeholder,
```

Comment:
should this be within bumping version?

Reply:
Renamed the MR to make it more clear that it includes more than just version bump.